### PR TITLE
First project: Hello-Microverse!

### PR DESCRIPTION
**Pull request by 3nityGitHub**
## In this PR,
- Installed linters for html and css
- Added .gitignore file
- Added index.html file with a h1 header
- Added style.css file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (ha-manel) [inline .gitignore]: - [ ] Great job with adding `node_modules` to `.gitignore` 👏 Kindly add `test.md` also as mentioned in the requirements 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
-# Hello-Microverse
-My first microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (ha-manel) [inline README.md:1]: - [ ] Your README file looks great however please stick to the new [template](https://github.com/microverseinc/readme-template) provided by Microverse 👍 Kindly copy all of the content then fill out the sections with information relevant to your project 💯

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @3nityGitHub👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Github flow is followed correctly ✔️ 

## Required Changes:

- [ ] From now on kindly choose the branch's name more carefully, it should describe the actions taken in the branch. Ex: "setup-project" 👍 

_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
```
Comment (GedeonTS) [inline .gitignore:1]: - [ ]  As requested by the previous reviewer, kindly make sure `test.md` is also in this  `.gitignore` file as it must be ignored by github

File: README.md [hunk 4]
```diff
@@ -1,2 +1,82 @@
...
+Deployment
+You can deploy this project using:
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/3nityGitHub)
+- Twitter: [@twitterhandle](https://twitter.com/tbarijohnson)
```
Comment (GedeonTS) [inline README.md]: - [OPTIONAL] I think It would be nice if you added your own handles here to make the README.md more descriptive.

**Review by GedeonTS — CHANGES_REQUESTED** [review]
#  STATUS : CHANGES REQUIRED :recycle: 

Hi @3nityGitHub , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  Linter checks are passing :+1: 
- Good use of github flow :+1: 

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @3nityGitHub   , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Pull request merged**